internal/stackdriver/stackdriver.go

```diff
@@ -43,6 +45,50 @@ func NewAPIClient(ctx context.Context, project string) (*API, error) {
 	}, nil
 }
 
+// RequestCount count returns the number of requests for the given offset and
+// query.
+func (a *API) RequestCount(ctx context.Context, query Query, offset time.Duration) (int64, error) {
+	logger := util.LoggerFromContext(ctx)
+	query = addFilterToQuery(query, "metric.type", requestCount)
+	endTime := time.Now()
+	startTime := endTime.Add(-1 * offset)
+	offsetString := fmt.Sprintf("%fs", offset.Seconds())
+
+	req := a.MetricClient.Projects.TimeSeries.List("projects/" + a.Project).
+		Filter(query.Query()).
+		IntervalStartTime(startTime.Format(time.RFC3339Nano)).
+		IntervalEndTime(endTime.Format(time.RFC3339Nano)).
+		AggregationAlignmentPeriod(offsetString).
+		AggregationPerSeriesAligner("ALIGN_DELTA").
+		AggregationGroupByFields("resource.labels.service_name").
+		AggregationCrossSeriesReducer("REDUCE_SUM")
+
+	logger.Debug("querying request count from Cloud Monitoring API")
```

Comment:
you might find it useful to add values here.
imagine we query latency for a service 3 times.
but you see same log 3 times, you won't show what the each query was for.
but if you add some specifics (e.g. reducer, or aligner), you can have a better time distinguishing them.

Reply:
Since reducer and aligner are very "abstract" concepts about how Cloud Monitoring works (and are always the same), what about using `intervalStartTime` and `intervalEndTime`, which are more useful and changed depending on when you call the function?